spec.bs

```diff
@@ -588,6 +589,14 @@ members:
         credential(s). The caller does not need to specify the
         [[#sctn-payment-extension-registration| payment extension]]; it is added
         automatically.
+
+    :  <dfn>localeHint</dfn> member
+    :: An optional list of **TODO, ISO-something?** locale descriptors, in
+        decreasing order of priority, that act as a hint from the website as to
+        what locale the users experience on that website is currently in.
```

Comment:
```suggestion
        the user's current locale on that site.
```

Reply:
(Used Addison' suggested language which changed this text, please take another look)